cardano-cli/src/Cardano/CLI/Options.hs

```diff
@@ -47,13 +50,21 @@ pref =
       , helpRenderHelp customRenderHelp
       ]
 
+-- The node related commands are shelley era agnostic for the time being.
+-- There is no need to guard them by the era argument.
+nodeCmdsTopLevel :: Parser ClientCommand
+nodeCmdsTopLevel =
+  AnyEraCommand . AnyEraCommandOf ShelleyBasedEraShelley . NodeCmds <$> pNodeCmds
```

Comment:
I think I'm not getting the full context of this change.
1. Why we're hardcoding a witness here?
2. "for the time being" - are we planning to make `node` command era based again?
3. If the witness isn't needed, why not do it like `ping` or `debug`?

Reply:
1. No longer applies
2. It's possible they add some requirement to running a node in the future, which would make the node related commands depends on the era
3. No longer applies